doc-src/SASS_CHANGELOG.md

```diff
@@ -23,6 +23,13 @@
   function now accepts a `$bracketed` parameter that controls whether the
   returned list has brackets.
 
+* Passing a string to `call($function-name, $args...)` indicating which function to invoke is
+  now deprecated. Instead pass callable returned from `function-reference($function-name)`.
```

Comment:
I saw a little of what you said about this in Slack, but what's the motivation behind calling this `function-reference()` instead of `function()` or something else shorter? It seems like the longer it is, the more difficult it'll be to convince people that it's better than passing the raw name.